eppo/src/main/java/cloud/eppo/android/dto/deserializers/RandomizationConfigResponseDeserializer.java

```diff
@@ -0,0 +1,194 @@
+package cloud.eppo.android.dto.deserializers;
+
+import static cloud.eppo.android.util.Utils.logTag;
+
+import android.util.Log;
+
+import com.google.gson.JsonDeserializationContext;
+import com.google.gson.JsonDeserializer;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParseException;
+
+import java.lang.reflect.Type;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+import cloud.eppo.android.EppoClient;
+import cloud.eppo.android.dto.Allocation;
+import cloud.eppo.android.dto.EppoValue;
+import cloud.eppo.android.dto.FlagConfig;
+import cloud.eppo.android.dto.OperatorType;
+import cloud.eppo.android.dto.RandomizationConfigResponse;
+import cloud.eppo.android.dto.ShardRange;
+import cloud.eppo.android.dto.TargetingCondition;
+import cloud.eppo.android.dto.TargetingRule;
+import cloud.eppo.android.dto.Variation;
+
+/**
+ *  Hand-rolled deserializer so that we don't rely on annotations and method names, which can be
+ *  unreliable when ProGuard minification is in-use and not configured to protect
+ *  JSON-deserialization-related classes and annotations.
+ */
+public class RandomizationConfigResponseDeserializer implements JsonDeserializer<RandomizationConfigResponse> {
+    private static final String TAG = logTag(EppoClient.class);
```

Comment:
copy pasta?

```suggestion
    private static final String TAG = logTag(RandomizationConfigResponseDeserializer.class);
```

Reply:
😱 Another good catch! THIS IS WHY WE DO PULL REQUESTS! 🙌